assets/openshift-apiserver/openshift-apiserver-deployment.yaml

```diff
@@ -72,6 +72,40 @@ spec:
 {{ end }}
       terminationGracePeriodSeconds: 90
       containers:
+{{- if .KonnectivityEnabled }}
+      - name: socks5-proxy
```

Comment:
We need to implement the lifecycle preStop which we have asked Red Hat to implement for HyperShift (see https://github.ibm.com/alchemy-containers/armada-update/issues/5065) and that you all implemented for OpenVPN (see https://github.ibm.com/alchemy-containers/openvpn-operator/pull/194).

Reply:
Thanks, done!